frontend/lib/norent/homepage.tsx

```diff
@@ -1,19 +1,364 @@
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("connection")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    Legal Protections
+                  </p>
+                  <p>
+                    Our letter cites the most up-to-date legal ordinances that
+                    protect tenant rights in your state.
+                  </p>
+                </div>
+              </div>
+              <div className="column is-one-third jf-has-centered-images">
+                <div>
+                  <StaticImage
+                    ratio="is-128x128"
+                    src={getImageSrc("letters")}
+                    alt="Chat boxes"
+                  />
+                </div>
+                <div>
+                  <p className="title is-size-4 has-text-weight-bold">
+                    Free Certified Mail
+                  </p>
+                  <p>
+                    After you’ve reviewed your letter, we send it to your
+                    landlord on your behalf by email and by certified mail.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <br />
+            <span className="is-hidden-mobile">
+              <BuildMyLetterButton />
+            </span>
+          </div>
+        </div>
+      </section>
+
+      <section className="hero">
+        <div className="hero-body">
+          <div className="container jf-tight-container has-text-centered jf-space-below">
+            <h2 className="title is-spaced">Legally vetted</h2>
+            <p className="subtitle is-size-4">
+              Our free letter builder was built with{" "}
+              <OutboundLink
+                href="https://www.justfix.nyc/about/partners"
+                rel="noopener noreferrer"
+              >
+                lawyers and non-profit tenants rights organizations
+              </OutboundLink>{" "}
+              across the nation to ensure that your letter gives you the most
+              protections based on your state.
+            </p>
+            <br />
+          </div>
+          <div className="container jf-wide-container">
+            <div className="columns is-mobile is-multiline is-variable is-8-desktop">
+              <div className="column is-one-fourth jf-has-centered-images">
+                <StaticImage
+                  ratio="is-128x128"
+                  src={getImageSrc("justfix", "png")}
+                  alt="Justfix logo"
+                />
+              </div>
+              <div className="column is-one-fourth jf-has-centered-images">
+                <StaticImage
+                  ratio="is-128x128"
+                  src={getImageSrc("nwbccc", "png")}
+                  alt="NWBCCC logo"
```

Comment:
All of these static images should probably have their alt text set to just the name of the partner (without the added "Logo"), and fully spelled-out, e.g. "Northwest Bronx Community and Clergy Coalition" rather than "NWBCC", since that's ultimately their semantic meaning.

Reply:
Also, you totally don't have to do this, but from a user's perspective, I wonder how many people will try clicking/tapping on these logos?  Makes me wonder if they should ink to the partner sites one day, might be worth bringing up w/ Tahnee.  Just a thought!